resources/views/travel/matrixitinerarypreview.blade.php

```diff
@@ -0,0 +1,18 @@
+@foreach($itinerary['itinerary']['slices'] as $slice)
+<div style="margin-bottom: 0.5rem; @if(! $loop->first && $loop->count > 1) margin-top: 1.5rem; @endif">{{ $slice['origin']['city']['name'] }} ({{ $slice['origin']['code'] }}) to {{ $slice['destination']['city']['name'] }} ({{ $slice['destination']['code'] }}) on {{ \Carbon\Carbon::parse($slice['departure'])->format('D, M j') }}</div>
+@foreach($slice['segments'] as $segment)
+<div style="display: grid; grid-template-columns: 5% 95%">
+    <img alt="{{ $segment['carrier']['shortName'] }}" src="https://www.gstatic.com/flights/airline_logos/35px/{{ $segment['carrier']['code'] }}.png" style="align-self: center;">
+    <div style="display: grid; grid-template-columns: 50% 25% 25%;">
+        <div style="display: block;"><strong>{{ $segment['origin']['city']['name'] }} ({{$segment['origin']['code']}}) to {{ $segment['destination']['city']['name'] }} ({{ $segment['destination']['code'] }})</strong> on {{ \Carbon\Carbon::parse($segment['departure'])->format('D, M j') }}</div>
+        <div style="display: block; grid-column: 2/span 2;"><strong>{{ \Carbon\Carbon::parse($segment['departure'])->format('g:i A') }}</strong> to <strong>{{ \Carbon\Carbon::parse($segment['arrival'])->format('g:i A') }}</strong> ({{ \Carbon\Carbon::parse($segment['arrival'])->diffForHumans(\Carbon\Carbon::parse($segment['departure']), \Carbon\CarbonInterface::DIFF_ABSOLUTE, true, 2) }})</div>
```

Comment:
can we render the year here too (also for arrival date)?

Reply:
Not sure what you meant by arrival date but I added the year to the two places we render a date here

Reply:
oops, yes, the return segment departure date. looks correct to me in Nova